types/onscan.js/index.d.ts

```diff
@@ -206,3 +206,33 @@ export interface OnScan {
 
 declare const onScan: OnScan;
 export default onScan;
+
+export interface Scan {
+    /**
+     * Quantity
+     */
+    qty: number;
+    /**
+     * Scanned code
+     */
+    scanCode: string;
+}
+
+export interface ScanEvent extends CustomEvent<Scan> {}
+
+export interface ScanError {
+    message: string;
+    scanCode: string;
+    scanDuration: number;
+    avgTimeByhar: number;
+    minLength: number;
+}
+
+export interface ScanErrorEvent extends CustomEvent<ScanError> {}
+
+declare global {
+    interface DocumentEventMap {
```

Comment:
Can we add `scanButtonLongPress` too? Not sure how exactly it fits in the `DocumentEventMap`, possibly try mapping it to undefined or never and see if either of those work.

Reply:
The documentation does mention it, but the parameters passed to the listeners are not documented, and I haven't found the ``scanButtonLongPress`` event anywhere in the source code. 🤔

It seems to me that the documentation is misleading and ``scanButtonLongPress`` just doesn't exist. 😕